int64 id_custom
geometry_msgs/PoseStamped pose_custom
float32 mark_ground_ms
float32 car_rot_ms
float32 pitch_ms
float32 roll_ms
float32 yaw_ms
